src/server/routes/forwardingRoute.ts

```diff
@@ -30,45 +35,53 @@ async function findNBNodeId(nodeId, lang) {
     return nodeId;
   }
 
-  const data = await resolveJsonOrRejectWithError(response);
+  const data = await resolveJsonOrRejectWithError<ExternalIds>(response);
 
   // The nodeId for language nb is the first item in externalIds array.
-  return data.externalIds[0];
+  return data!.externalIds[0];
```

Comment:
Skulle denne vore ?

Reply:
Den kan godt være det, tenkte bare jeg skulle holde meg nærmest mulig tidligere funksjonalitet i og med at jeg ikke var sikker på hvordan dette skulle testes. Denne kalles uansett bare innenifra en try/catch, så jeg regner med at det går fint. 

Hvis jeg kan få en liten innføring i hvordan disse endepunktene brukes kan vi skrive en mer bærekraftig versjon senere.

Reply:
Denne er vel her for å støtte gamle ndla-urler med format ndla.no/node/id der id er det samme som du kan finne i article-api med oldNdlaUrl. Disse er importert fra gamle drupal-baserte ndla.no. Du kan teste ved å søke opp artikler i ed med status Importert. Disse har dette feltet, og dermed kan du teste å hente artikkel basert på denne iden.

Reply:
OK. Skal jeg ta en omskrivning i denne, eller skal vi ta det senere?

Reply:
Treng ikkje det.